zio-kafka/src/main/scala/zio/kafka/utils/ExtraZStreamOps.scala

```diff
@@ -0,0 +1,40 @@
+package zio.kafka.utils
+
+import zio._
+import zio.stream._
+
+object ExtraZStreamOps {
+
+  implicit class ZStreamOps[R, E, A](val stream: ZStream[R, E, A]) extends AnyVal {
```

Comment:
`final` and `private` are missing
```suggestion
  implicit final class ZStreamOps[R, E, A](private val stream: ZStream[R, E, A]) extends AnyVal {
```

See for example: https://github.com/zio/zio/blob/bdc08ea21dce1485e9280ba76ee9d74d8e0b202f/core/shared/src/main/scala/zio/ZIO.scala#L5131

Reply:
Okay, this is against what I learned in the past. O well, times change...